apps/dashboard/app/views/layouts/nav/_featured_apps.erb

```diff
@@ -14,6 +16,9 @@
                 target: link.new_tab? ? "_blank" : nil
               ) do
             %>
+              <%- if rendering_pinned_apps && g.apps.size > Configuration.pinned_apps_menu_length -%>
+                </span>(<%= app_index+1 %>/<%= g.apps.size %>)</span>
+              <%- end -%>
```

Comment:
Having the (1/6) => (5/6) prefix makes the menu very busy - it is a lot of text to try to ignore when you want to just read the names of the apps.

I would add the count to the header instead. i.e.

![screen 2021-02-17 at 1 04 02 PM](https://user-images.githubusercontent.com/512333/108247535-c3a20700-7120-11eb-9ab8-9bf9a08c5d0f.png)

or

![screen 2021-02-17 at 1 03 40 PM](https://user-images.githubusercontent.com/512333/108247556-cac91500-7120-11eb-91d0-bf9dc8a1906e.png)



Also it is a simpler change because you are only changing the header, you don't need the bulk of the changes in this commit to accomplish it.

Reply:
Done!